### PR TITLE
improve logging

### PR DESCRIPTION
**Pull request by msach22**

### CODE REVIEW

File: src/OTSession.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ export default class OTSession extends Component {
     const sessionEvents = sanitizeSessionEvents(this.props.eventHandlers);
     setNativeEvents(sessionEvents);
     this.createSession();
-    logOT(this.props.apiKey, this.props.sessionId);    
+    logOT(this.props.apiKey, this.props.sessionId, 'rn_initialize');
```
Comment (JayTokBox) [inline src/OTSession.js]: I would prefer OTLog instead of logOT. This way you can package all OT* (sdk + your own)

File: src/helpers/OTHelper.js [hunk 2]
```diff
@@ -19,19 +19,26 @@ const reassignEvents = (type, customEvents, events) => {
 
 const sanitizeBooleanProperty = property => (property || property === undefined ? true : property);
 
-const logOT = (apiKey, sessionId) => {
+const getLogBody = (apiKey, sessionId, action, connectionId) => {
```
Comment (JayTokBox) [inline src/helpers/OTHelper.js]: I was thinking HTML body here , when I first read it.
Can we just say getLog?

File: src/OTSession.js (hunk 1 above)
Comment (msach22) [reply src/OTSession.js]: `OTLog` does not follow camel case and it's not global so I think it's fine as is.

File: src/helpers/OTHelper.js (hunk 2 above)
Comment (msach22) [reply src/helpers/OTHelper.js]: updated in latest commit

**Pull request merged**